views/js/qtiCreator/tpl/forms/response/responseForm.tpl

```diff
@@ -22,6 +22,18 @@
     </select>
 </div>
 
+<div class="panel">
+    <label for="" class="has-icon">{{__ "Response baseType"}}</label>
```

Comment:
I know that it is defined in the task, but what if we can make this label more human? Now it looks like just a description of the field, which is bad UX practice. `Response base type` looks much better. An explanation can be and should be provided in the tooltip.

Reply:
Done :)